### PR TITLE
added Arabidopsis thaliana VCF sample

### PR DESCRIPTION
**Pull request by brunocontrerasmoreira**
Hi, I have been testing the VEP script on plants and found out there was no suitable sample VCF file.
I am adding one here from https://europepmc.org/article/MED/27293186 for Arabidopsis thaliana

### CODE REVIEW

File: examples/arabidopsis_thaliana.TAIR10.vcf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+##fileformat=VCFv4.1
+# source: intersection_10001.vcf.gz
+# https://europepmc.org/article/MED/27293186
+1	135	.	C	T	28	PASS	DP=5	GT:GQ:DP	1|1:28:5
```
Comment (dglemos) [inline examples/arabidopsis_thaliana.TAIR10.vcf:4]: Could you add the header line names? Something like: 
`#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE`

File: examples/arabidopsis_thaliana.TAIR10.vcf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+##fileformat=VCFv4.1
+# source: intersection_10001.vcf.gz
```
Comment (dglemos) [inline examples/arabidopsis_thaliana.TAIR10.vcf:2]: Thanks for adding a new file.
Here should be two '##' without space between 'source'. The same in L3.

Comment (brunocontrerasmoreira) [reply examples/arabidopsis_thaliana.TAIR10.vcf]: I have fixed it following your suggestions, thanks!

**Comment by brunocontrerasmoreira** [comment]
I have done as requested @dlemos, obrigado

**Comment by dglemos** [comment]
Thanks for the changes!

**Pull request merged**